frontend/src/components/modals/Modals.styled.ts

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { Wrapper } from "../wrapper/Wrapper.styled";
+
+const ModalsWrapper = styled(Wrapper)``;
```

Comment:
This wrapper is using only in modals, rename it to "ModalsWrapper" and import it and use instead, this inherit is unnecessary in that case.

Reply:
ok, done